results/latex/pegasus-test-extrinsic-100.tex

```diff
@@ -0,0 +1,18 @@
+\newcommand{\pegasusFbsOracleFactual}{59\%}
```

Comment:
@anton164 I'm guessing we never created this file because the annotations aren't complete. Would it be better to be updating the 25, 50, or 75 set?

Reply:
pretty sure we're using the pegasus-75 set!